<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowVivod</class>
 <widget class="QWidget" name="WindowVivod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Вывести</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Итоговые параметры фигуры:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>угол B^C:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showBC">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSplitter" name="splitter_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>угол C^A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showCA">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>радиус:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>сторона C:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showC">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSplitter" name="splitter_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>угол D^A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showDA">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>сторона D:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showD">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>угол A^B:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showAB">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>периметр:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>сторона A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showA">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>сторона B:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showB">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSplitter" name="splitter_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>угол C^D:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showCD">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSplitter" name="splitter_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>площадь:</string>
       </property>
      </widget>
      <widget class="QLabel" name="showS">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
